crates/utils/src/utils/markdown/image_links.rs

```diff
@@ -60,7 +62,8 @@ fn find_urls<T: NodeValue + UrlAndTitle>(src: &str) -> Vec<(usize, usize)> {
   ast.walk(|node, _depth| {
     if let Some(image) = node.cast::<T>() {
       if let Some(srcmap) = node.srcmap {
-        let (_, node_offset) = srcmap.get_byte_offsets();
+        let (_, node_offset) = dbg!(srcmap.get_byte_offsets());
+        dbg!(node_offset, image.url_len(), image.title_len());
```

Comment:
markdown-it normalizes markdown links with urlencode, so in case of special characters the length changes and this calculation is not correct anymore.